types/ui-select/index.d.ts

```diff
@@ -1,19 +1,77 @@
-// Type definitions for ui-select 0.13.2
+// Type definitions for ui-select 0.19.8
```

Comment:
Changes based on:
* https://github.com/angular-ui/ui-select/tree/v0.19.8
* https://github.com/angular-ui/ui-select/wiki/Accessing-$select

Reply:
Just as an FYI, the patch version is not necessary as it will get ignored by the bot. Details here: https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library